catalog-info.yaml

```diff
@@ -97,6 +97,10 @@ spec:
         trigger_mode: "none"
```

Comment:
Same - these need to go to main

Reply:
I've added this change for main in https://github.com/elastic/connectors/pull/2893